.eslintignore

```diff
@@ -1,8 +1,10 @@
-# Contains the PDF file of the Tag as JSON string, thus does not need to be linted
-src/components/CheckIn/tagTemplate.ts
```

Comment:
Why was this removed? The file is very big.

Reply:
I deeply regret mistakenly removing the file and sincerely apologize for my oversight. I was unaware of its size and significance and will take immediate steps to rectify it.

Reply:
Please restore the files in the file. It's a learning process